generators/python-v2/ast/src/Reference.ts

```diff
@@ -70,4 +71,8 @@ export class Reference extends AstNode {
     public getFullyQualifiedModulePath(): string {
         return this.modulePath.join(".");
     }
+
+    public getCompletePath(): string {
+        return `${this.getFullyQualifiedModulePath()}.${this.name}`;
+    }
```

Comment:
nit: Thoughts `getFullyQualifiedPath`?

Reply:
This helper had existed, but yep happy to change as part of this PR.